### DISCUSSION TITLE
Netbox loading Slow whille accessing on Intranet.

### DISCUSSION THREAD

**Discussion by Raunak1620** [netbox-community/netbox#17226] [Q&A]
Hi Buddies,  
I have deployed netbox dev server on rhel 8 hosted in one of our office building and accessing it from another building windows desktop. However i am able to access but i am facing extreme slowness in accessing. Can somebody help me on this. I will not be able to use if this will be this slow.

**Comment by Raunak1620** [comment]
it's running locally,  nginx will help me to solve this???

**Comment by a084ed22** [comment]
This is dependent on your setup and/or environment and/or network. There's not much that can be done without knowing how that is pieced together for you.
